xbmc/cores/VideoPlayer/VideoRenderers/ColorManager.cpp

```diff
@@ -0,0 +1,511 @@
+/*
+ *      Copyright (C) 2016 Lauri Mylläri
+ *      http://kodi.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include <math.h>
+#include <string>
+
+#include "system.h"
+#include "ColorManager.h"
+#include "cores/VideoPlayer/VideoRenderers/RenderFlags.h"
+#include "filesystem/File.h"
+#include "settings/Settings.h"
+#include "utils/log.h"
+#include "utils/TimeUtils.h"
+
+using namespace XFILE;
+
+CColorManager::CColorManager() :
+  m_hProfile(NULL)
```

Comment:
I missed these while working with the lcms2 features and previous fixes. I found a few similar issues - I'll make sure to test with lcms2 disabled before requesting another review.